beeload/README.md

```diff
@@ -0,0 +1,63 @@
+# beeload
+
+GitHub Actions bundle to upload Pull Requests to Swarm.
+
+## Getting Started
+
+To run this action:
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
See, you have it right in your example ;-)

Reply:
Yup, doesn't work though 🙂 

See: https://github.com/filoozom/swarm-actions/runs/5447761098

In `Run /./beeload` -> `Run peter-evans/create-or-update-comment@v1`